app/controllers/tasks_controller.rb

```diff
@@ -8,12 +9,17 @@ def show
     @task_name = @task.name
     @task_description = @task.description
 
+    #checks if task is completed, and that date is past
     @completed = false
-
-    if @task.completed_date != nil
-      @task_completed_date = @task.completed_date
-      @completed_date =  @task_completed_date.strftime("%m/%d/%Y")
-      @completed = true
+    @future_date = false
```

Comment:
I think you could clean up your `@future_date` logic here, since it seems like it is being set a one additional unnecessary time. I think you could get this down to one single if statement to set all of the variables you have here.